app/admin.go

```diff
@@ -177,6 +178,10 @@ func (a *App) TestSiteURL(siteURL string) *model.AppError {
 	if err != nil || res.StatusCode != 200 {
 		return model.NewAppError("testSiteURL", "app.admin.test_site_url.failure", nil, "", http.StatusBadRequest)
 	}
+	defer func() {
+		io.Copy(ioutil.Discard, res.Body)
+		res.Body.Close()
```

Comment:
1/5. By using 
```suggestion
		_, _ = ioutil.ReadAll(r.Body)
		_ = r.Body.Close()
```
the code would make clear that any errors are ignored on purpose.

Reply:
done